testsuite/Main.hs

```diff
@@ -573,6 +573,7 @@ runClashTest = defaultMain $ clashTestRoot
       , clashTestGroup "SynthesisAttributes"
         [ NEEDS_PRIMS_GHC(outputTest ("tests" </> "shouldwork" </> "SynthesisAttributes") allTargets [] [] "Simple" "main")
         , NEEDS_PRIMS_GHC(outputTest ("tests" </> "shouldwork" </> "SynthesisAttributes") allTargets [] [] "Product" "main")
+        , NEEDS_PRIMS_GHC(outputTest ("tests" </> "shouldwork" </> "SynthesisAttributes") allTargets [] [] "InstDeclAnnotations" "main")
```

Comment:
`NEEDS_PRIMS` / `NEEDS_PRIMS_GHC` may not be needed here (they only change things under the experimental new evaluator). Can you remove it here please in case it's not needed (it won't affect the CI on this branch, I can fix it on the evaluator branch if necessary)?

Reply:
It is not needed and I have removed it.